### PR TITLE
Created summary component

### LINKED ISSUE #66: Add summary for number type

### PR DESCRIPTION
**Pull request by MichaelDysart**
Created summary of number question type
Close #66

### CODE REVIEW

**Review by rinadina — APPROVED** [review]
Looks fine.

**Review by JacksonSchoe — APPROVED** [review]
Looks good Michael

File: src/main/js/App.js [hunk 1]
```diff
@@ -195,12 +196,25 @@ const App = () => {
         .catch(console.log);
     };
 
+    const summariseSurvey = () => {
+        console.log(`${webUrl}retrieveSurvey?id=${userSurveyId}`);
```
Comment (TaylorAbraham) [inline src/main/js/App.js:195]: Lingering console.log

File: src/main/js/Summary.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    OPEN_ENDED: "openEnded",
+    NUMERICAL: "numberQuestion",
+    DROPDOWN: "dropdown",
+};
+
+/*
+ * The frontend core which renders all subcomponents for surveys
+ */
+const Summary = (props) => {
+    console.log(props);
```
Comment (TaylorAbraham) [inline src/main/js/Summary.js:15]: And here

**Review by TaylorAbraham — CHANGES_REQUESTED** [review]
Looks great. Nice job adding the histogram. Just remove a couple console.logs

**Review by rinadina — APPROVED** [review]

File: src/main/js/Summary.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+    return (
+        <div className="qq-summary">
+            {props.questions.map((q, i) => {
+                switch(q.type) {
+                    case qType.OPEN_ENDED:
+                        return (
+                            <div className="qq-app mv" key={i} label="Open Question Summary">
+                                <div>{`Question ${i + 1} - Numerical`}</div>
```
Comment (rinadina) [inline src/main/js/Summary.js]: Is it suppose to say Numerical for open ended ?

**Pull request merged**